Command/RemoveCacheCommand.php

```diff
@@ -20,56 +21,89 @@
+resolved filter(s), outputting results using the following basic format:
+  <info>image.ext[filter] (removed|skipped|failure)[: (image-path|exception-message)]</>
 
-Paths should be separated by spaces:
-<info>php app/console %command.name% path1 path2</info>
-All cache for a given `paths` will be lost.
+<comment># bin/console %command.name% --filter=thumb1 foo.ext bar.ext</comment>
+Remove cache for <options=bold>both</> <comment>foo.ext</comment> and <comment>bar.ext</comment> images for <options=bold>one</> filter (<comment>thumb1</comment>), outputting:
+  <info>- foo.ext[thumb1] removed</>
+  <info>- bar.ext[thumb1] removed</>
 
-If you use --filters parameter:
-<info>php app/console %command.name% --filters=thumb1 --filters=thumb2</info>
-All cache for a given filters will be lost.
+<comment># bin/console %command.name% --filter=thumb1 --filter=thumb3 foo.ext</comment>
+Remove cache for <comment>foo.ext</comment> image using <options=bold>two</> filters (<comment>thumb1</comment> and <comment>thumb3</comment>), outputting:
+  <info>- foo.ext[thumb1] removed</>
+  <info>- foo.ext[thumb3] removed</>
 
-You can combine these parameters:
-<info>php app/console %command.name% path1 path2 --filters=thumb1 --filters=thumb2</info>
+<comment># bin/console %command.name% foo.ext</comment>
+Remove cache for <comment>foo.ext</comment> image using <options=bold>all</> filters (as none were specified), outputting:
+  <info>- foo.ext[thumb1] removed</>
+  <info>- foo.ext[thumb2] removed</>
+  <info>- foo.ext[thumb3] removed</>
 
-<info>php app/console %command.name%</info>
-Cache for all paths and filters will be lost when executing this command without parameters.
 EOF
             );
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->setupOutputStyle($input, $output);
+        $this->outputCommandHeader();
+
+        [$images, $filters] = $this->resolveInputFiltersAndPaths($input);
+
+        foreach ($images as $i) {
+            foreach ($filters as $f) {
+                $this->runCacheImageRemove($i, $f);
+            }
+        }
+
+        $this->outputCommandResult($images, $filters);
+
+        return $this->getResultCode();
+    }
+
+    private function runCacheImageRemove(string $image, string $filter): void
     {
-        $paths = $input->getArgument('paths');
-        $filters = $input->getOption('filters');
+        if (!$this->outputMachineReadable) {
+            $this->io->text(' - ');
+        }
+
+        $this->io->group($image, $filter, 'blue');
+        $this->io->space();
 
-        if (empty($filters)) {
-            $filters = null;
+        if ($this->cacheManager->isStored($image, $filter)) {
+            $this->cacheManager->remove($image, $filter);
+            $this->io->status('removed', 'green');
```

Comment:
not sure about the colors. removed could be red i feel...

Reply:
in this case, I guess it make sense green because what you're trying to do is actually to remove cache, so it worked fine

Reply:
you are probably right. red could make people think that there was a problem with those files.